FindPairsDiff.py

```diff
@@ -0,0 +1,20 @@
+#Time complexity: O(n)
+#Space complexity: O(n)
+def func(nums,target):
+    res=[]
+    dict1={}
+    i=0
+    while i < len(nums):
+        while i<len(nums) and nums[i] in dict1:
+            i+=1
+        if i==len(nums):
+            break
+        if nums[i]+target in dict1:
+            res.append((nums[i],nums[i]+target))
+        if nums[i]-target in dict1:
+            res.append((nums[i],nums[i]-target))
+        dict1[nums[i]]=1
+        i+=1
+    return res
```

Comment:
The output should be an integer, why are you returning the array? Please redo your code and submit it for review.